module/VuFind/config/module.config.php

```diff
@@ -683,7 +683,7 @@
                 'class' => \Doctrine\ORM\Mapping\Driver\AttributeDriver::class,
                 'cache' => $doctrineCacheType,
                 'paths' => [
-                    'module/VuFind/src/VuFind/Db/Entity',
+                    __DIR__ . '/../src/VuFind/Db/Entity',
```

Comment:
This isn't required with the current configuration, but I noticed that with certain configurations the ColocatedMappingDriver balked at the path.